CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unversioned
 
 - Major: Added customizable shortcuts. (#2340)
+- Minor: Added an option to show animated SevenTV Avatar rather than twitch Avatar.
```

Comment:
```suggestion
```
Changelog entries are not being added in this fork. As when it gets merged into upstream, it will simply be one changelog entry as "Major: Added 7TV support"